connectionpool_systems_test.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -e
+
+function run_tests() {
```

Comment:
instead of having another test runner, would it not just be easier to add the `conn_pool` tag to the `integration.sh` runner?